README.md

```diff
@@ -20,10 +23,16 @@ DC/OS Docker is designed to optimize developer cycle time. For a more production
 - A recent kernel that supports Overlay FS
 - git
 
-### Mac
+### macOS (Experimental)
```

Comment:
prefer just `### Mac` with a bold line after saying it's experimental and why. That way the page anchor link doesn't change later.